teamcity-rest-client-impl/src/test/kotlin/org/jetbrains/teamcity/rest/testUtils.kt

```diff
@@ -19,7 +19,7 @@ fun setupLog4jDebug() {
     Logger.getLogger("org.apache.http").level = Level.ERROR
 }
 
-val publicInstanceUrl = "https://teamcity.jetbrains.com"
+val publicInstanceUrl = "http://localhost:8111"
```

Comment:
How server on this port will be started? Do you plan to include scripts which start it to this repository or it will be provided by TeamCity build configuration which will run these tests?

Reply:
The script for starting up TC server instance will be in the build configuration step. Also, it requires access to private artifactory to pull the test data so that it would work within internal network only.

Reply:
So we won't be able to automatically run tests for external pull requests in this repository? This will be unfortunate.

Reply:
Ok, I'll check that options do we have here.